performanceMetrics/src/App/App.js

```diff
@@ -150,14 +151,14 @@ const App = (props) => {
 		let componentMetrics = [], promises = [];
 
 		for (let buildDate of listOfTestDates) {
-			promises.push(axios.get('./develop/' + buildDate + '/' + selectedComponent + '.txt'));
+			promises.push(fetch('./develop/' + buildDate + '/' + selectedComponent + '.txt').then(result => result.text()));
 		}
 
 		Promise.allSettled(promises).then((results) => {
-			const successfulResults = results.filter((result) => result.status === 'fulfilled');
+			const successfulResults = results.filter((result) => result.value.includes('ReactVersion'));
```

Comment:
Note: `fetch` resolves promises as fullfield even if the data is not correct. I replaced this filter method in order to get only the correct data.
ex: 
![image](https://user-images.githubusercontent.com/63335823/224685553-8db862a1-8a9c-4d4d-9fb5-189ca8f008b5.png)